bin/start_wifi_connect.sh

```diff
@@ -2,7 +2,12 @@
 
 IS_CONNECTED=''
 
-if [[ $DEVICE_TYPE =~ ^(pi1|pi2)$ ]]; then
+if [[ $DEVICE_TYPE = "pi1" ]]; then
+    echo "The latest version of WiFi Connect does not support Raspberry Pi 1. Exiting..."
```

Comment:
WiFi connect doesn't care what Pi it is running on. It cares if it has a WiFi adapter. You can easily add this using USB. Maybe it's a fine temporary workaround to use this logic, but the *proper* fix is to check if the device has a WiFi interface. 

You can use `nmcli` to detect this:

Device without WiFi adapter:
```bash
$ nmcli device wifi list
$ 
```

Device with WiFi adapter:
```bash
$ nmcli device wifi list
IN-USE  BSSID              SSID                  MODE   CHAN  RATE        SIGNAL 
[...]
```

Since we already rely on Network Manager on both Raspbian and Balena, that seems like a reasonable dendency.

Reply:
I agree, thanks. That seems pretty neat.